core/src/main/scala/kafka/admin/ConsumerGroupCommand.scala

```diff
@@ -635,10 +630,11 @@ object ConsumerGroupCommand extends Logging {
         val resetPlanPath = opts.options.valueOf(opts.resetFromFileOpt)
         val resetPlanCsv = Utils.readFileAsString(resetPlanPath)
         val resetPlan = parseResetPlan(resetPlanCsv)
-        partitionsToReset.map { topicPartition =>
-            if (resetPlan.keySet.contains(topicPartition))
-              (topicPartition, resetPlan(topicPartition))
-            else null
+        resetPlan.keySet.map { topicPartition =>
```

Comment:
Adding a comment here for lack of a better place. It seems we still require either --all-topics or --topic to use the CSV option?